ftw/contentpage/behaviors/configure.zcml

```diff
@@ -12,7 +12,7 @@
         title="ftw.contentpage content categories behavior"
         description="Extends a content by the content_categories field"
         provides="ftw.contentpage.behaviors.content_categories.IContentCategories"
-        factory="plone.behavior.AnnotationStorage"
+        factory=".content_categories.ContentCategoriesStorage"
```

Comment:
@jone Nope - I use the annotation blabla impl (https://github.com/4teamwork/ftw.contentpage/pull/211/files#diff-a5b966409543b61ce8841f4c2305986bR42). The dexterity annotation factory is using the same.